packages/workspace/src/parser/parse.ts

```diff
@@ -55,18 +55,7 @@ export const parse = async (
   functions: Functions = {},
 ): Promise<Required<ParseResult>> => {
   const srcString = naclFile.toString()
-  const [patchedSrc, elements, sourceMap, errors] = await parseBuffer(
-    srcString,
-    filename,
-    functions
-  )
-  const fixedErrors = filterErrors(errors, patchedSrc)
-    .map(error => generateErrorContext(srcString, error))
-    .map(error => restoreErrorOrigRanges(patchedSrc, error))
-    .map(error => ({ severity: 'Error' as SaltoErrorSeverity, ...error }))
-  return {
-    elements: elements.map(flattenElementStr),
-    sourceMap,
-    errors: fixedErrors,
-  }
+  return process.env.USE_NEW_PARSER
```

Comment:
NEW is only new for a limited time 😄 

```suggestion
  return process.env.USE_INTERNAL_PARSER
```

Reply:
👍